pkg/manifests/manifests.go

```diff
@@ -2064,10 +2065,10 @@ func (f *Factory) MetricsServerDeployment(apiAuthSecretName string, kubeletCABun
 	// are rotated.
 	dep.Spec.Template.Annotations["monitoring.openshift.io/serving-ca-secret-hash"] = hashByteMap(servingCASecret.Data)
 
-	// Hash the metrics client cert and propagate it as an annotation to the
-	// deployment's pods to trigger a new rollout when the metrics client cert
+	// Hash the metrics server client cert and propagate it as an annotation to the
+	// deployment's pods to trigger a new rollout when the metrics server client cert
 	// is rotated.
-	dep.Spec.Template.Annotations["monitoring.openshift.io/metrics-client-cert-hash"] = hashByteMap(metricsClientCert.Data)
+	dep.Spec.Template.Annotations["monitoring.openshift.io/metrics-server-client-cert-hash"] = hashByteMap(metricsServerClientCert.Data)
```

Comment:
If we could add an `s` at the end `metrics-server-client-certs`, it'd be great.

Reply:
> I think we'll need to do this in two steps

Great catch, iiuc you're right. At least this wouldn't break anything, we'd just end up with both annotations in upgraded clusters.

Reply:
Since this was tech preview it wont have a problem with upgrade when moves GA right as there is no upgrade support in techpreview?

IIUC if it was GA in 4.15 then we should do it in two steps?

Reply:
Yes, the metrics-server has been GA for a while. Perhaps some clusters had upgraded back then and will be upgrading again when the metrics-server is GA ;). This is just to ensure they have clean resources, as Jan mentioned. If you think it’s negligible and not worth it, we can certainly ignore this.

Reply:
ya no problem in adding but I was curious if that has impact or not since it was not GA :)

Reply:
(had been GA for 3 days though ;) Your call)